frontend/src/metabase/query_builder/components/NativeQueryEditor/CodeMirrorEditor/CodeMirrorEditor.module.css

```diff
@@ -155,5 +155,55 @@
         text-overflow: ellipsis;
       }
     }
+
+    /* Syntax highlighting */
```

Comment:
I believe this is an incorrect merge resolution. These lines are already removed in https://github.com/metabase/metabase/pull/54253

Reply:
That might very well be the case.

Reply:
Fixed with https://github.com/metabase/metabase/pull/52436/commits/0f1a2cc366e2c497c8e8f2759ad831928868b1fd